index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="pl-PL">
+	<head>
+		<link rel="stylesheet" type="text/css" href="css/style.css">
+		<script src="http://code.jquery.com/jquery-1.10.2.js"></script>
```

Comment:
add tag meta with charset, title to head and move js files before closing body tag